acid/templates/base.html

```diff
@@ -18,6 +18,7 @@
           href="{{ url_for('static', filename='bootstrap-4.1.1-dist/css/bootstrap.min.css') }}">
     <link rel="stylesheet"
           href="{{ url_for('static', filename='main.css') }}">
+    {% block meta_content %}{% endblock %}
```

Comment:
I don't like the name *meta_content* because inside `<head></head>` you can put more then just meta tags. You already have there `<link>` and `<title>` and there could be more.
I would like it to be more generic like for example: `extra_head` or `heder`.